src/Assets/index.js

```diff
@@ -0,0 +1,3 @@
+import homepage from "./Images/homepage.jpg";
```

Comment:
Suggestion: For better clarity **homepage** can be as **homePageImg** as it is image.